unittests/test_containers.py

```diff
@@ -13,7 +13,8 @@
     'Sarus', 'Sarus+nocommand', 'Sarus+nopull', 'Sarus+mpi', 'Sarus+load',
     'Shifter', 'Shifter+nocommand', 'Shifter+mpi', 'Shifter+nopull',
     'Shifter+load',
-    'Singularity', 'Singularity+nocommand', 'Singularity+cuda'
+    'Singularity', 'Singularity+nocommand', 'Singularity+cuda',
+    'Apptainer', 'Apptainer+nocommand', 'Apptainer+cuda'
```

Comment:
It looks like these params are in alphabetical order, but it's fine to leave like that

Reply:
I was thinking to change them but in the end I left it like that to be side by side with the Singularity one